examples/tokens/token_airdrop_transaction_cancel.py

```
@@ -163,9 +173,9 @@ def cancel_airdrops(client, operator_key, pending_airdrops):
         cancel_airdrop_receipt = cancel_airdrop_tx.execute(client)
 
         if cancel_airdrop_receipt.status != ResponseCode.SUCCESS:
-            print(f"Failed to cancel airdrop: "
-                  f"Status: {cancel_airdrop_receipt.status}"
-                 )
+            print(
+                f"Failed to cancel airdrop: " f"Status: {cancel_airdrop_receipt.status}"
```

Ruff (0.14.10)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)